components/Navbar/Navbar.tsx

```diff
@@ -10,7 +10,7 @@ export const navigationLinks = [
   { name: "O firmie", path: paths.solidgate },
   { name: "Galeria", path: paths.galeria },
   { name: "Kontakt", path: paths.kontakt },
-  // { name: "Blog", path: paths.blog },
+  { name: "Blog", path: paths.blog },
```

Comment:
can we change Read more button to "zobacz więcej" or smth?

Reply:
![Uploading Screenshot 2022-04-03 at 12.23.36.png…]()

Reply:
oh we use polish :) 
okey now it's "Czytaj dalej..."